src/main/scala/Builders.scala

```diff
@@ -62,26 +88,66 @@ trait EnrichedTextBuilder {
   def withObjectTitle(x: ObjectTitle): this.type = withEnrichedText(x)
   def withPronunciation(x: Pronounce): this.type = withEnrichedText(x)
   def withVirtualLocation(x: Virtloc): this.type = withEnrichedText(x)
-  def withXml(x: NodeSeq): this.type = withContent(dataRecord(x))
-  def withText(x: String): this.type = withContent(dataRecord(x))
 
   protected def withEnrichedText[T <: EnrichedTextOption : CanWriteXML](x: T): this.type = withContent(dataRecord(x))
   protected def withContent(x: DataRecord[_]): this.type
 }
 
+trait MixedContentBuilder {
+  def withText(x: String): this.type = withContent(dataRecord(x))
+
+  /** Appends the given XML to the model object.
+    * Note that this method is _not_ type-safe!
+    * __No__ validation is performed to verify that the XML matches the expected schema..
+    *
+    * @deprecated Use multiple invocations of other methods to construct the data for a type-safe approach
+    */
+  @deprecated("Use multiple invocations of other methods for a type-safe approach", since = "3.x.3")
+  def withXml(x: NodeSeq): this.type = withContent(dataRecord(x))
+
+  protected def withContent(x: DataRecord[_]): this.type
```

Comment:
It looks like the implementation is the same every time this trait is extended. Worth providing it as a default implementation?

Reply:
I wish I could! Because the implementation calls `.copy`, I don't think it's possible to do this other than with a macro.

Do you know of another way to do it?

Reply:
I saw it too late 😩 I suppose you could define an abstract class inheriting from `Builder` and `MixedContentBuilder`? That may not be very elegant though...

Reply:
Even if I did this, I still won't be able to implement `withContent`. That's because case classes' `.copy()` is not defined in a trait.